source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -122,6 +110,98 @@ async def fetch_page(
         yield result.paging.next.after
 
 
+async def fetch_page_incremental(
+    cls: type[CRMObject],
+    http: HTTPSession,
+    log: Logger,
+    log_cursor: LogCursor,
+) -> AsyncGenerator[CRMObject | LogCursor, None]:
+    # Runs V3 API queries and formats the execution to match
+    # the CDK's fetch_changes requirements 
+    assert isinstance(log_cursor, datetime)
+
+    url = f"{HUB}/crm/v3/objects/{cls.PROPERTY_SEARCH_NAME}"
+
+    input = {
```

Comment:
Can we use the `log_cursor` value here to initialize `input["after"]`, so that it doesn't start over from the beginning each round?